user.go

```diff
@@ -386,6 +377,17 @@ func (user *User) Connect() bool {
 		user.log.Warnln("Error connecting to WhatsApp:", err)
 		return false
 	}
+	if user.bridge.Config.Bridge.HistorySync.Backfill && !user.historySyncLoopsStarted {
+		go user.handleHistorySyncsLoop()
+		user.historySyncLoopsStarted = true
+
+		if user.bridge.Config.Bridge.HistorySync.BackfillMedia && user.bridge.Config.Bridge.HistorySync.EnqueueBackfillMediaNextStart {
+			var priorityCounter int
+			for _, portal := range user.bridge.GetAllPortalsForUser(user.MXID) {
+				user.EnqueueMediaBackfills(portal, &priorityCounter)
+			}
+		}
+	}
 	return true
```

Comment:
Hmm, actually not entirely sure if this is the best place either, since it's technically not fully connected yet (auth happens after connecting). Doing it on `*events.Connected` might be safest.